python/py-pysal/Portfile

```diff
@@ -4,28 +4,52 @@ PortSystem              1.0
 PortGroup               python 1.0
 PortGroup               github 1.0
 
-github.setup            pysal pysal 1.14.4 v
+github.setup            pysal pysal 23.07 v
 revision                0
-name                    py-pysal
+name                    py-${github.project}
+github.tarball_from     tarball
```

Comment:
why not `archive`? If it needs to be `tarball` then you can omit it, as that is the current default in the `github` PG